src/utils/prompt.ts

```diff
@@ -1,14 +1,15 @@
 import * as vscode from 'vscode';
 
 export function name(filename: string): Promise<string | undefined> {
+	const value = filename.replace(/.\w+$/, '-copy.$1');
```

Comment:
It looks like we need to add an escape for the point here.

Otherwise, it will match anything at the end: `test.d?ts` → match `?ts`.

Reply:
Also, I think, we ca remove `-copy` suffix at all. Of course, if you wanted.

Reply:
Thanks for reviewing. PR has been updated.

I think `-copy` is OK, it helps avoid warning overwirte existing file. Otherwise, the file name is selected by default, uses just type a new name, I think it's fine.